lib/shoryuken/processor.rb

```diff
@@ -22,7 +22,7 @@ def process
         end
       end
     rescue Exception => ex
-      logger.error { "Processor failed: #{ex.message}" }
+      logger.error { "Processor failed: Exception: #{ex.class.name} with message:#{ex.message}" }
```

Comment:
That might be related to the error tracking I mentioned above, I would strongly recommend an error tracking for tracking errors. The `raise` below would eventually be logged in an error tracking, along with the error message and message body. 

I'm not against adding `Exception:`  though, but if we do it in here, we may want to add it [here](https://github.com/phstc/shoryuken/blob/c7a0984fbb6811a307b5c062047ad1cacbfd41b9/lib/shoryuken/manager.rb#L104) and [here](https://github.com/phstc/shoryuken/blob/bc7938942270a1171d1597064698d88c3f36a4ec/lib/shoryuken/middleware/server/exponential_backoff_retry.rb#L26) too - all the places that `ex.backtrace.join("\n")`.